README.md

```diff
@@ -19,6 +19,7 @@ Realistically, this could be compressed into a helper script, but those steps we
 |cacls|cacls [filepath]|lists user permissions for the specified file, wildcards supported|
 |driversigs|driversigs| enumerate installed services Imagepaths to check the signing cert against known edr/av vendors|
 |ipconfig|ipconfig| Simply gets ipv4 addresses, hostname and dns server|
+|ldapsearch|ldapsearch| Executes LDAP searches |
```

Comment:
For the Middle column please add the arguments following the [name] for required and [opt: name] for optional similar to how the others commands are handled